torch_xla/csrc/xla_graph_executor.cpp

```diff
@@ -358,6 +359,23 @@ void XLAGraphExecutor::SyncTensorsGraph(std::vector<XLATensorPtr>* tensors,
   }
 }
 
+std::string XLAGraphExecutor::SyncTensorsGraphDumpHlo(
+    std::vector<XLATensorPtr>* tensors, absl::Span<const std::string> devices,
+    bool sync_ltc_data) {
+  tsl::profiler::TraceMe activity("SyncTensorsGraphDumpHlo",
+                                  tsl::profiler::TraceMeLevel::kInfo);
+  SyncTensorsConfig config;
+  config.sync_ltc_data = sync_ltc_data;
```

Comment:
you don't really need to pass `sync_ltc_data`, since you are not updating the IR.